staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -405,6 +421,57 @@ func (t *tracker) add(gvr schema.GroupVersionResource, obj runtime.Object, ns st
 	return nil
 }
 
+type objectVersioner struct{}
```

Comment:
objectVersioner is copied and modified from [here](https://github.com/kubernetes/kubernetes/blob/2601aadbb6cc7a4da0e91ceb74a021752ba7cf07/staging/src/k8s.io/client-go/tools/cache/mutation_cache.go#L239)